sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -413,6 +425,39 @@ public String getId() {
         return queryItemsInternal(new SqlQuerySpec(query), new CosmosQueryRequestOptions(), classType);
     }
 
+    /**
+     * Initializes the container by warming up the caches and connections.
+     * <br>The execution of this method is expected to result in some RU charges to your account.
+     * The number of RU consumed by this request varies, depending on data consistency, size of the overall data in the container,
+     * item indexing, number of projections. For more information regarding RU considerations please visit
+     * <a href="https://docs.microsoft.com/en-us/azure/cosmos-db/request-units#request-unit-considerations">https://docs.microsoft.com/en-us/azure/cosmos-db/request-units#request-unit-considerations</a>.
+     * @return Mono of Void
```

Comment:
Please add a comment on how customers should handle eventual errors - with the new logic below any retry after receiving an error on the first attempt would be a no-op.

Which means customers have two options:
1) On error crash the process - any retry would happen by trying to restart the app/service on a new process (I assume this would be the most common case - after call this method only makes sense if being called during app initialization before the health check/watch dog signal to the load balancer that it is able to accept incoming requests

2) Swallow the exception and continue (without preinitialized caches just hope lazy load will work)

Thanks